src/resources/postcss/common-responsive.pcss

```diff
@@ -5,28 +5,28 @@
  * @todo @juanfra: Check this once we define https://github.com/the-events-calendar/tribe-common/pull/1379
  */
 
-@import "utilities/_all.pcss";
+/* Import Mixins */
+@import '_mixins.pcss';
```

Comment:
where do the variables come from if we remove this?

Reply:
from common-skeleton and common-full